chapters/scoping.tex

```diff
@@ -317,7 +317,7 @@ \section{Simultaneous Inner/Outer Declarations}\label{simultaneous-inner-outer-d
   der(x) = if isEnabled then -x else 0;
 end ConditionalIntegrator;
 
-model SubSystem "subsystem that 'enable' its conditional integrators"
```

Comment:
This has the same grammatical error that was corrected above: 'enable' -> 'enables'

Reply:
I believe the apostrophes should be removed so 'enable' -> enables.
It seems they are used to indicate Modelica code in comments below, but it doesn't make sense in this place as there is no variable named enable.

Reply:
@HansOlsson I have additionally pushed to this PR but is was merged before. How can I restart this PR? Or shall I create a new PR?

Reply:
A new PR is likely simplest.